styles/style.css

```diff
@@ -36,7 +28,19 @@ button {
 
 /* Buttons */
 /* INSERT BUTTON STYLES HERE */
+button.volunteer-button {
+	background-color: rgb(255, 255, 255);
+	border-style: none;
+	font-size: 20px;
+}
 
+button.donate-button {
```

Comment:
This styling is great, but you need to add the `volunteer-button` class to the `button` element in your HTML.